src/types/realtime-ai/typeOverrides.d.ts

```diff
@@ -152,10 +147,13 @@ export abstract class Transport {
   abstract sendReadyMessage(): void;
   abstract getAllMics(): Promise<MediaDeviceInfo[]>;
   abstract getAllCams(): Promise<MediaDeviceInfo[]>;
+    abstract getAllSpeakers(): Promise<MediaDeviceInfo[]>;
```

Comment:
Totally naive question: This looks like a lot of duplication from `@pipecat-ai/client-js`. Are all of these overrides or could we actually use the types from the `@pipecat-ai/client-js`?

Also, totally minor point, the directory for this file is still called `realtime-ai`. Maybe that should change now that realtime-ai is no longer a thing?

Reply:
We don't need to change the directory because what actually matters is the path inside tsconfig:
https://github.com/pipecat-ai/rtvi-client-react-native-daily/blob/0.3.0/tsconfig.json#L6.

Unfortunately, we cannot override only specific types in this case. Some core types, like `MediaDeviceInfo`, `MediaStreamTrack`, are provided by `react-native-webrtc` instead of the browser, so we must override everything. We do the same with `react-native-daily-js` and `daily-js`.

Reply:
But I can rename the repo only in order to keep consistency. 👍

Reply:
Done.